CoEpi/ui/alerts/AlertsViewController.swift

```diff
@@ -7,26 +7,70 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 
 class AlertsViewController: UIViewController {
+    private let viewModel: AlertsViewModel = .init()
```

Comment:
The view models should still be injected, since they can have more dependencies.